### PR TITLE
Fetch speaker list from the network

### PR DESCRIPTION
**Pull request by BoD**
It was fetched from the cache first

### CODE REVIEW

File: shared/data/build.gradle.kts [hunk 1]
```diff
@@ -51,7 +51,8 @@ apollo {
 
     introspection {
       schemaFile.set(file("src/commonMain/graphql/schema.graphqls"))
-      endpointUrl.set("https://androidmakers-2023.ew.r.appspot.com/graphql")
+      endpointUrl.set("https://confetti-app.dev/graphql")
```
Comment (martinbonnin) [inline shared/data/build.gradle.kts:54]: Good catch 👍

File: shared/data/build.gradle.kts [hunk 2]
```diff
@@ -51,7 +51,8 @@ apollo {
...
+      headers.set(mapOf("conference" to "androidmakers2024"))
```
Comment (martinbonnin) [inline shared/data/build.gradle.kts:55]: Is this required for introspection? I would hope the schema does not depend on the conference

**Review by martinbonnin — APPROVED** [review]

File: shared/data/build.gradle.kts (hunk 2 above)
Comment (BoD) [reply shared/data/build.gradle.kts:55]: No but the IJ plugin also looks at this headers and inject them when running queries from the UI :)

Comment (martinbonnin) [reply shared/data/build.gradle.kts:55]: I see. Well this is a bit surprising. Might adding a comment?

**Pull request merged**